src/document/DocumentCommandHandlers.js

```diff
@@ -589,9 +589,18 @@ define(function (require, exports, module) {
         if(isFolder) {
             extension = "";
         }
-
-        var suggestedName = baseFileName + "-" + (_nextUntitledIndexToUse++) + extension,
+		
+		if (_nextUntitledIndexToUse === 1)
+		{
```

Comment:
Also please follow the code conventions used in the rest of the file: no `{` on its own line, it's on the same line as the `if` keyword and conditional